lib/src/html_renderer.dart

```diff
@@ -104,6 +111,11 @@ class HtmlRenderer implements NodeVisitor {
   @override
   void visitText(Text text) {
     var content = text.textContent;
+
+    if (_tagfilterEnabled) {
```

Comment:
It seems this should be implemented in the parsers. It adds complexity to do anything but the most simple translation of the HTML nodes here to text. (I'm not crazy about the special line-splitting logic below, for example...)

Reply:
OK I think this makes sense.